apps/admin-gui/src/assets/i18n/en.json

```
@@ -1635,7 +1635,9 @@
       "SUCCESSFULLY_CREATED": "Sponsored member was successfully created",
       "USER": "User",
       "WAS_CREATED": "was successfully created",
-      "COPY_INFORMATION": "Please save this information. It will not be possible to access this information later."
+      "COPY_INFORMATION": "Please save this information. It will not be possible to access this information later.",
+      "PASSWORD_RESET": "Send activation link to entered email",
```

Since you have changed this for the bulk method, I think this would make sense to have a similar format. 
```suggestion
      "PASSWORD_RESET": "Let the created user change his password via a given email address",
```